homeassistant/components/caldav/coordinator.py

```diff
@@ -18,6 +21,25 @@
 OFFSET = "!!"
 
 
+async def async_get_calendars(
```

Comment:
Nothing in this function seems to involve the coordinator. I'd put it in another module.